docs/getting-started.md

```diff
@@ -96,7 +96,7 @@ analyzer. To override this check use the `--allow-dynamic-versions` option:
 ```bash
 $ cli/build/install/ort/bin/ort analyze -i [mime-types-dir] -o [analyzer-output-dir] --allow-dynamic-versions
 The following package managers are activated:
-        Gradle, Maven, SBT, NPM, Yarn, GoDep, PIP, Bundler, PhpComposer, Stack
+        Bundler, Composer, GoDep, Gradle, Maven, NPM, PIP, SBT, Stack, Yarn
```

Comment:
The previous ordering was based on the used programming language. IMO this makes more sense than using an alphabetical ordering here.

Reply:
The previous text was from long ago when ORT seemingly used a different ordering for this output. Nowadays ORT uses alphabetical ordering, so I preferred to make the ordering match the (current) reality.